helm-chart/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: Nextcloud AIO Helm Chart
 description: A generated Helm Chart for Nextcloud AIO from Skippbox Kompose
-version: 4.0.1
-apiVersion: v1
+version: 4.2.0
+apiVersion: v2
```

Comment:
@szaimen looks good. I've sent https://github.com/kubernetes/kompose/pull/1567 upstream and it got merged, so next kompose release should be v2 by default.

Reply:
all right! Thanks a lot! :)